qa/tasks/salt.py

```diff
@@ -177,6 +177,8 @@ def setup(self):
         self.__set_debug_log_level()
         self.sm.start_master()
         self.sm.start_minions()
+        self.sm.enable_minions()
+        self.sm.enable_masters()
```

Comment:
instead of `enable_masters` (plural) could we do `enable_master` (singular) like we have for `start_master`, above?